core/chains/evm/monitor/balance.go

```diff
@@ -39,7 +39,7 @@ type (
 		ethClient      evmclient.Client
 		chainID        *big.Int
 		chainIDStr     string
-		ethKeyStore    keystore.Eth
+		ethKeyStore    txmgr.KeyStore[gethCommon.Address, *big.Int, gethTypes.Transaction, int64]
```

Comment:
same thing here...just a gentle reminder that we're only worried about propagating the generics to the txm. we don't need to worry about other components yet?